nowcast/tests/test_constants.py

```diff
@@ -0,0 +1,21 @@
+"""This test is to ensure the constants are not accidentally altered."""
```

Comment:
This is a little bit [Frog And Toad Try Not To Eat Cookies](https://sites.fas.harvard.edu/~fs22k/Assignments/frog-toad.html) but I'm not sure there's a better way to do it so 🤷

Reply:
i guess what we're protecting against is conceptually no different than someone changing the implementation of smoothing or something and us not catching it, which an output validation integration test would capture, so we could move to that once it's ready.